client/app/scripts/liveblog-edit/module.js

```diff
@@ -216,12 +216,13 @@ define([
                 var deferred = $q.defer();
                 notify.info(gettext('saving blog settings'));
                 var changedBlog = {
-                        blog_preferences: vm.blogPreferences,
-                        original_creator: vm.original_creator._id,
-                        blog_status: vm.blog_switch === true? 'open': 'closed'};
-                    angular.forEach(vm.newBlog, function(value, key) {
-                        changedBlog[key] = value;
-                    });
+                    blog_preferences: vm.blogPreferences,
+                    original_creator: vm.original_creator._id,
+                    blog_status: vm.blog_switch === true? 'open': 'closed'
+                };
+                angular.forEach(vm.newBlog, function(value, key) {
```

Comment:
underscore `_.create` is a magic, fast and awesome function.
But I see you don't use lodash or underscore yet, so maybe is better to use `angular.extend` instead.